sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,50 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometer</EFDB_Topic>
+    <Description>LabJack connected accelerometer unit.</Description>
```

Comment:
I'm a little confused how the ESS xml works in general.
We'll have ~3 systems of ~3 accelerometers or so.
The xml appears to be for a single accelerometer? So where is the "indexing" of the different devices/accelerometers?

Reply:
Depending on the answer to those questions the Description could use some more detail.

Reply:
ok, based on slack discussion, this is probably fine, but maybe consider:
Single accelerometer device connected via LabJack

Reply:
I improved the description.